facts.txt

```diff
@@ -141,4 +141,8 @@
         "The average age of a French oak tree harvested for use in creating wine barrels is 170 years."
         "10,000 varieties of wine grapes exist worldwide."
     },
+	"Egg": {
```

Comment:
comma is missing in every line.

Reply:
Are you talking about within the curly braces? I am just following the convention i see in the file.